demo/src/main/java/hu/flowacademy/stockmarket/persistance/model/Stock.java

```diff
@@ -0,0 +1,115 @@
+package hu.flowacademy.stockmarket.persistance.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+
+
+@Entity
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+@Table(name = "stocks")
+public class Stock {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id", updatable = false, nullable = false)
```

Comment:
nem fontos a `@Column`, mivel az id alapvetően beállítja ezeket a dolgokat

Reply:
Javítva.